libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -153,9 +169,30 @@ void AC_PrecLand::init()
     }
 }
 
+// inertial_buffer - create/update inertial buffer
+void AC_PrecLand::inertial_buffer()
+{
+    // If PLND_LATENCY parameter is not within bounds, return without doing anything
+    if ( _inertial_buffer_length <= 0 || _inertial_buffer_length > 250 ) {
+        return;
+    }
+
+    // calculate inertial buffer size from delay length (loop runs at 400hz)
+    _inertial_buffer_size = (int)roundf(_inertial_buffer_length / (1000/400.0));
```

Comment:
_inertial_buffer_size = (uint16_t)roundf(_inertial_buffer_length / (1000/400.0f));

Reply:
Fixed in commit, thanks for pointing out!